### ISSUE TITLE
Provide keyboard and screen reader users with a way to move focus to a notification

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/vaadin-notification#121]
### Description
Notifications appear in the at the "bottom" of the page's DOM, making them difficult to reach via keyboard. Thus, keyboard-only users, including those using screen readers, need to tab through the entire page to reach the notification. This is especially problematic for notifications with interactive content like buttons.

### Expected outcome
Keyboard-only users can move focus to notifications through a standardized shortcut. (Not sure if there are any appropriate a11y standards for this?)

### Actual outcome
Keyboard-only users need to tab through the entire focusable DOM to reach notifications.

**Comment by knoobie** [comment]
@rolfsmeds you probably looking for aria live region. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions 

Sadly it's not the easiest to implement because the container (aria live region) has to be in the initial HTML response of a page, otherwise must screenreader won't announce it. (See the note in mozillas doc)

**Comment by rolfsmeds** [comment]
I looked into the live regions spec when I created this ticket, but couldn't find anything related to moving keyboard focus to the region. My understanding is that a live region is announced without moving focus to it.

**Comment by knoobie** [comment]
You are right, you don't have to move the focus to the notification thanks to the live region. ;) 

If possible, you should not interrupt the flow of the user. Keyboard only users should be able to see the notification and screen reader users should get the notification announced. Otherwise you have the same problem with notification like with e.g. the dialog - where the focus has to return to the origin element once the notification is closed or the user wants to go back.
